src/server/src/Server.cpp

```diff
@@ -140,7 +155,6 @@ bool Server::updateModule()
     return true;
 }
 
-
```

Comment:
The adopted convention is having two spaces between each function, can you please put it back? :)

Reply:
done